### DISCUSSION TITLE
control.LayerSwitcher

### DISCUSSION THREAD

**Discussion by OriginalSin** [openglobus/openglobus#776] [Q&A] [answered]
control.LayerSwitcher - почему то отключено добавление слоев?

**Comment by Zemledelec** [comment]
LayerSwitcher был подвергнут cleanup'у, потому что код был сложноватыый в поддержке, и было бы лучше переписать его с использованием ui утилит из папки ./ui. Я создал задачу, https://github.com/openglobus/openglobus/issues/777

**Comment by Zemledelec** [comment, marked as answer]
New LayerSwitcher was added.
